assets/src/setup/pages/save/index.js

```diff
@@ -192,7 +192,7 @@ export function Save() {
 							target="_blank"
 							rel="noreferrer"
 						>
-							{ __( 'Visit your site', 'amp' ) }
+							{ __( 'Visit', 'amp' ) }
```

Comment:
Maybe it would be more clear if the button were placed _below_ the iframe? There could be a “Browse” and a “Customize” button right under the preview. This could be displayed _instead_ of the “Live view of a page on your site…” text, as the buttons should give that context. 

I was originally thinking that the Dashboard had a “Customize” button but it actually says “Customize Your Site”:

![image](https://user-images.githubusercontent.com/134745/86495205-321a8c00-bd2d-11ea-9b80-3cac4249fd6a.png)

Nevertheless, if the buttons were under the iframe then this should make this clear.

Also, the buttons would be placed near the Previous/Finish buttons which keep the CTAs in the same area.

Reply:
I like that approach. It would look something like this then:

![image](https://user-images.githubusercontent.com/16200219/86495493-28167e80-bd69-11ea-8edc-c8c5a8b3b310.png)

Reply:
Updated in b2661a832